pygfx/renderers/wgpu/_blender.py

```diff
@@ -132,7 +132,7 @@ def get_shader_code(self, blender):
             @location(1) pick: vec4<u32>,
         };
         fn get_fragment_output(depth: f32, color: vec4<f32>) -> FragmentOutput {
-            if (color.a < 1.0) { discard; }
+            // if (color.a < 1.0) { discard; }
```

Comment:
I guess the color interpolation accuracy in the fragment shader will lead to alpha not exactly equal to 1.0.
I tried to change the threshold to 0.99, and the result was also correct. 

But I think maybe they should not be discarded directly here, just treat them as opaque.
Or we may need a threshold attribute like "alpha_test" to control the discarding here.

Reply:
It's important that (semi)transparent fragments are discarded here, otherwise the depth buffer will be set, and the blending will be incorrect. We could lower the threshold, but I want to be really sure that is necessary.

Is this what fixes #295? Do you know why? It's not obvious to me at first glance.

Reply:
> Is this what fixes #295? Do you know why? It's not obvious to me at first glance.

Yes, It is. 

When using "vertex color", the pixel color in the fragment shader will be calculated according to the interpolation of three vertices of the triangle. I guess there will be a floating-point precision problem.  Although the alpha value of all the three vertices color is 1.0, it may not be exactly equal to 1.0 after interpolation,  there will be precision error. At this time, some pixels may meet the alpha < 1.0 and be discarded.
Looks like that we need to keep this line of code and slightly lower the threshold.

Reply:
Can we define a constant for this?